src/UI/PlayerSetupInfo.java

```diff
@@ -25,14 +26,38 @@
   @Override
   public String toString()
   {
-    return String.format("%s %s %s %s %s %s", currentCo, currentColor, currentFaction, flipUnits, currentTeam, currentAi);
+    StringBuilder coString = new StringBuilder();
+    // If we have more than one CO, build a parenthetical list within our list
+    if( coList.size() > 1 )
+      coString.append("( ");
+    for( int co : coList )
+      coString.append(co).append(" ");
+    if( coList.size() > 1 )
+      coString.append(")");
```

Comment:
I probably would have just put this all in a single `if( coList.size() > 1)` for clarity, but this works too.